website/content/en/preview/getting-started/getting-started-with-karpenter/scripts/add-monitoring.sh

```diff
@@ -1,22 +0,0 @@
-#!/bin/bash
```

Comment:
Should we remove these files from the versioned sections

Reply:
Updated